pkg/gds/admin_test.go

```diff
@@ -1613,7 +1613,6 @@ func (s *gdsTestSuite) TestResend() {
 		"vaspID": vaspErrored,
 	}
 	actual := &admin.ResendReply{}
-	sent := time.Now()
```

Comment:
They were, apparently the new version of `staticcheck` does a better job dealing with test files -- these lines were never used. I did talk to @pdeziel who mentioned that we might want to create different variables for these so that the email timestamps match in the assertions below. I made that change and am merging now, thanks for the review!